bash_completion.sh

```diff
@@ -426,10 +426,15 @@ _buildtest ()
     unittests)
       local opts="--coverage --help --pytestopts --sourcefiles -c -h -p -s"
 
+      COMPREPLY=( $( compgen -W "${opts}" -- $cur ) )
+      ;;
+    tutorial-examples)
+      local opts=""
```

Comment:
i think here you can just put `local opts="-h --help"`

Reply:
@shahzebsiddiqui I tried this but still doesn't seem to work.

Reply:
yeah this was easy fix i made the correction now see the latest commit https://github.com/buildtesters/buildtest/pull/1307/commits/f8bcf7c0a6d494f1bae79c7d9bf8a8b08b513c85

The change i made does the tab completion for the command which is done at the very bottom case statement 
![Screen Shot 2022-12-06 at 2 48 00 PM](https://user-images.githubusercontent.com/12942230/206008201-25c6dcae-68b4-4706-ae0d-6402c2106f0a.png)


The one that you were working on was tab completion on the options after typing `buildtest tutorial-examples`

![Screen Shot 2022-12-06 at 2 48 39 PM](https://user-images.githubusercontent.com/12942230/206008330-7f661eef-7555-46df-98af-fce3c47722b9.png)